packages/SystemsThinkingWorkbench-Core.package/STWPlayfield.class/instance/addSimulationButtons.st

```diff
@@ -0,0 +1,15 @@
+initialization
+addSimulationButtons
+	
+	self playButton: (STWButton new
+		contents: 'Start';
+		clickHandler: [ self toggleSimulation. ]).
+	self addMorph: self playButton.
+	self playButton position: self position.
```

Comment:
Interestingly, it can. From experiences in the previous project i remember the owner would have an influence on how position: would behave.